core/src/replay-parse/replay-parse.mod.resolver.ts

```diff
@@ -72,4 +77,14 @@ export class ReplayParseModResolver {
         await this.rpService.enableSubscription(submissionId);
         return this.pubsub.asyncIterator(submissionId);
     }
+
+    // TODO GraphQL type
```

Comment:
Can we type this?

Reply:
[2a57b0a](https://github.com/SprocketBot/sprocket/pull/155/commits/2a57b0a626ecb9cc5c85bbd7215c8d4627d5466f)